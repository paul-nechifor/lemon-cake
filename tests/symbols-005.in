(list (quote a) (quote s) (quote d) (quote f) 'x' (list 1 (quote ab)))
