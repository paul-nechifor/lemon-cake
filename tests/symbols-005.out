(a s d f 'x' (1 ab))
